iquip/explorer.py

```diff
@@ -0,0 +1,33 @@
+"""
+App module for showing the experiment list and opening an experiment.
+"""
+
+from typing import Optional, Tuple
+
+from PyQt5.QtCore import QObject
+from PyQt5.QtWidgets import QLabel, QVBoxLayout, QWidget
+
+from qiwis import BaseApp
+
+class ExplorerFrame(QWidget):
+    """Frame for showing the experiment list and opening an experiment."""
+    def __init__(self, parent: Optional[QObject] = None):
+        """Extended."""
+        super().__init__(parent=parent)
+        # widgets
+        self.testLabel = QLabel("explorer", self)
+        # layout
+        layout = QVBoxLayout(self)
+        layout.addWidget(self.testLabel)
+
+
+class ExplorerApp(BaseApp):
+    """App for showing the experiment list and opening an experiment."""
+    def __init__(self, name: str, parent: Optional[QObject] = None):
```

Comment:
The same applies here as well as in `ExplorerFrame`.

Reply:
Umm.. I think the remaing is more appropriate in `ExplorerApp`.

The reasons are
1. `BaseApp` is a subclass of `QObject` and `QObject` has a `QObject` parent, not `QWidget`.
Please refer to [QObject constructor](https://doc.qt.io/qt-6/qobject.html#QObject).
2. In `createApp()` of `Qiwis`, we set parants of apps as the `Qiwis` instance, and it is also `QObject`.

What do you think of it?

Reply:
You are right, of course the apps and `Qiwis` are not `QWidget`s. They are `QObject`s which can use `PyQt5`'s signal and slot system.
We should keep using `QObject` for the `parent` of apps.

Reply:
Please just make a blank line instead!